doc/pipeline.rst

```diff
@@ -821,10 +821,32 @@ combined:
   deletions (0 copies) (``ampdel``).
 - Confidence interval overlapping zero (``ci``).
 - Standard error of the mean (``sem``), a parametric estimate of confidence
-  intervals which behaves similarly.
+  intervals which behaves *similarly*.
 
 In each case, adjacent segments with the same value according to the given
 criteria are merged together and the column values are recalculated
 appropriately. Segments on different chromosomes or with different
 allele-specific copy number values will not be merged, even if the total copy
 number is the same.
+
+
+.. _bintest:
+
+``bintest``
```

Comment:
First of all, regarding where to put this: since `bintest` is being called as part of the batch command, I think the best place is here, in `pipeline.rst`, even if it's quite extensive already.

By the way, speaking of the batch, could you also please add bintest to the list of commands above, in the “The pipeline executed by the batch command is equivalent to...” section?

Reply:
Adding `bintest` to this section, I also added missing steps for ".call.cns" generation and several points regarding [tricky post-processing part](https://github.com/etal/cnvkit/blob/25d91cdb962639c98d041c3b05c7de659c41dc70/cnvlib/batch.py#L195-L211):
- `segmentrics --ci` is run with alpha=0.5 (instead of default=0.05=5% as usually done in statistics). Do you have an idea why ?
- `segmetrics.do_segmetrics()` has no `skip_low=False` flag as argument
=> Possible `--drop-low-coverage` is only applied when running `cnvkit.py segmetrics` alone, [within `commands.py`](https://github.com/etal/cnvkit/blob/25d91cdb962639c98d041c3b05c7de659c41dc70/cnvlib/commands.py#L1324-L1325)
=> Maybe we should add this arg to `segmetrics.do_segmetrics()` and sync it with `batch --drop-low-coverage` for consistency ?
- As some "files" are only produced internally and not written, I simulated them adding a ".tmp" suffix

==> To sum up, I am not very confident about what I wrote here ! (commit afbd0f7ff1a4dc21cc8917f9cdc1f4154a758904)

Reply:
Adding `--drop-low-coverage` to `segmetrics` is a good idea, and it should match `batch` and the other commands.

`segmetrics` uses a narrow confidence interval for the default filtering approach in `batch` because wide confidence intervals can lead to some pathological behavior -- the end effect is to merge adjacent segments with similar log2 values or a lot of noise in their probes. It's not the same as the p-value alpha. A default of 0.1 might be reasonable but I haven't benchmarked it, so 0.5 is the "safe" approach to only filter out very noisy segments in the final calling stage of the pipeline.